pkg/ext-proc/main.go

```diff
@@ -156,3 +191,25 @@ func main() {
 	s.Serve(lis)
 
 }
+
+var onlyOneSignalHandler = make(chan struct{})
+
+// SetupSignalHandler registered for SIGTERM and SIGINT. A context is returned
+// which is cancelled on one of these signals. If a second signal is caught,
+// the program is terminated with exit code 1.
+func SetupSignalHandler() context.Context {
```

Comment:
what is the difference between this and the setup we have above for signal handling?

Reply:
We also make a call to the controller runtime's version of SetupSignalHandler at line 158, I believe we should have one call, perhaps only to this function.